docs/DESIGN.md

```diff
@@ -222,24 +229,24 @@ The server must be configured with a certificate and corresponding private key.
 This certificate will be presented to clients who wish to negotiate TLS
 connections.
 
-The server will load trusted CA certs from the environment in the usual 
-operating-system defined location. These CA certs will be used to validate 
-the trust chain of the certificate presented by the client. Since the client 
-certificate will be used as the basis of authentication, the server MUST NOT 
-be configured to trust CA roots that are not trusted to verify client 
-identities for purposes of authentication.
+The server will load the CA roots to use for client certificate
+validation from an explicit location specified by the user (see CLI UX).
```

Comment:
For the record I think this is better too. It's easier to tighten/reason about the system this way.